connections/angular.json

```diff
@@ -25,6 +25,7 @@
               "src/assets"
             ],
             "styles": [
+              "@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
and we need to make sure we can work around this, with the TW Colors, so it overrides